UI/css/main.css

```diff
@@ -0,0 +1,407 @@
+body {
+    margin: 0;
```

Comment:
Avoid repeating your self. For example the `margin:0` and `padding:0` rules are repeated in body and h3. It would have been better if you put common rules under a group of selectors like
```css
body, h3 {
   margin: 0;
   padding: 0;
}
```
By doing this, you only have one place to update if you decide to make changes. Another way of doing it would be creating a class which you would add elements that would use such styles

Reply:
Otherwise. Looks Good to be merged

Reply:
Cool. Let me work on your suggestions and merge my work